cost/aws/object_storage_optimization/aws_object_storage_optimization.pt

```diff
@@ -657,7 +661,7 @@ policy "policy_s3_object_list" do
 end
 
 ###############################################################################
-# Escalation
+# Escalations
 ###############################################################################
 
 escalation "esc_email" do
```

Comment:
Line 703 and Line  752 - add a space after the comma so it looks like so: `raise join($$errors,"\n")`

Same Line 828 - should be: `path join(["/api/governance/projects/", rs_project_id, "/applied_policies/", switch(ne(meta_parent_policy_id, ""), meta_parent_policy_id, policy_id)])`